src/gluonts/evaluation/_base.py

```diff
@@ -300,6 +299,9 @@ def get_metrics_per_ts(
             "abs_target_mean": self.abs_target_mean(pred_target),
             "seasonal_error": seasonal_error,
             "MASE": self.mase(pred_target, median_fcst, seasonal_error),
+            "MASE_strict": self.mase_strict(
+                pred_target, median_fcst, seasonal_error
+            ),
```

Comment:
Could we actually have just one implementation of the `mase` method maybe, i.e. the one from the book, and handle any adjustment here? If I'm not wrong, the adjusted MASE we have implemented just makes MASE = 0 whenever the seasonal error is too small, so this could be done here with an `if` statement

Reply:
Sure, this makes sense, however what should we present as MASE then? The custom version or the original one?

Reply:
That should work and i've implemented this in the latest commit

Reply:
Probably we would want to do this to all of the other metrics with similar flag handling?